CMakeLists.txt

```diff
@@ -192,11 +194,11 @@ if(NOT MSVC)
   # Check if our libraries have unresolved symbols
   #if(NOT APPLE AND NOT HPX_WITH_APEX)
   if(NOT APPLE AND NOT WIN32 AND NOT PHYLANX_WITH_SANITIZERS)
-    phylanx_add_link_flag_if_available(-Wl,-z,defs TARGETS SHARED EXE)
+    hpx_add_link_flag_if_available(-Wl,-z,defs TARGETS SHARED EXE)
```

Comment:
This shouldn't be necessary anymore, the missing cmake macro was added here: 72117850cc7154b20edc7e5397cfb50f78d84853.